modules/services/variables.tf

```
@@ -48,6 +52,7 @@ locals {
   cert_manager_version        = var.cert_manager_version
   letsencrypt_issuer_email    = var.letsencrypt_issuer_email
   jaeger_host                 = "opentelemetry-collector.${local.otel_namespace}.svc.cluster.local"
+  trace_sampel_pct            = var.trace_sample_pct
```

Looks like a typo here:

```suggestion
  trace_sample_pct            = var.trace_sample_pct
```